src/easyreflectometry/calculators/calculator_base.py

```diff
@@ -158,14 +158,22 @@ def remove_item_from_model(self, item_id: str, model_id: str) -> None:
         """
         self._wrapper.remove_item(item_id, model_id)
 
-    def fit_func(self, x_array: np.ndarray, model_id: str) -> np.ndarray:
+    def reflectity_profile(self, x_array: np.ndarray, model_id: str) -> np.ndarray:
         """Function to perform a fit.
 
         :param x_array: points to be calculated at
         :param model_id: The model id
         """
         return self._wrapper.calculate(x_array, model_id)
 
+    def fit_func(self, x_array: np.ndarray, model_id: str) -> np.ndarray:
+        """Function to perform a fit.
```

Comment:
This does not actually perform a fit, does it?
It only calculates the profile based on the current set of parameters in the model.

Reply:
removed fit_function from calculator_base.  Adjusted docstring